config/default/metrics_service.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     app.kubernetes.io/name: mintmaker
     app.kubernetes.io/managed-by: kustomize
   name: controller-manager-metrics-service
-  namespace: system
```

Comment:
In our deployment, namespace is set in kustomization.yaml files, so it will be changed by kustomize. But it doesn't hurt to change this.